src/code/views/link-relation-view.coffee

```diff
@@ -167,21 +167,30 @@ module.exports = LinkRelationView = React.createClass
           @renderScalarPulldown(@state.selectedScalar)
         )
       )
-      (div {className: 'graph', id:'relation-graph'},
-        (Graph {xAxis: source, yAxis: target, link: @props.link, graphStore: @props.graphStore})
-      )
-      if @props.link.relation.isCustomRelationship and not @props.link.relation.customData?
-        (div {className: 'graph-hint'},
-          (span {}, "#{tr "~NODE-RELATION-EDIT.CUSTOM_HINT"} ")
+      (div {className: 'bottom'},
+        (div {className: 'graph', id:'relation-graph'},
+          (Graph
+            xAxis: source
+            yAxis: target
+            link: @props.link
+            graphStore: @props.graphStore
+          )
         )
-      (div {className: 'fixed-bottom'},
+      )
+
+      (div {className: 'bottom'},
```

Comment:
This change intentionally changes the fixed-bottom nature of the text box.  Dan thought it was better to have predictable spacing between the because box and graph, as per this story:  https://www.pivotaltracker.com/story/show/133221703  -- in cases where variable names are long the graph and because box would overlap.  Dan thought it would be better to scroll in such instances.